### ISSUE TITLE
client_max_body_size does not apply to HTTP

### ISSUE THREAD

**Issue by frafra** [geopaparazzi/GSS#37]
Hi :)
`client_max_body_size` works only when connecting to HTTPS, not when using HTTP (or doing HTTPS offloading in front of the service):

https://github.com/geopaparazzi/GSS/blob/961272148430f522c01d418492a77b0b1bb80ece/docker/https/nginx.conf#L55

**Comment by moovida** [comment]
Yeah, should be removing that one anyway. Not yet sure how to handle large mapsforge files though.

**Comment by frafra** [comment]
The default `client_max_body_size` is `1M`, which is rather small. It could help checking if compression is enabled.

There are different ways to handle very large file uploads. 

Large files can be uploaded using [TUS](https://tus.io) and/or [S3 multipart uploads](https://aws.amazon.com/blogs/aws/amazon-s3-multipart-upload/).

There is a [client for DART](https://pub.dev/packages/tus_client_dart) and some [support for Django](https://github.com/alican/django-tus/), but they do not seem to be well maintained. We mostly use [Uppy](https://uppy.io/) as an interface to upload data to a S3 storage via TUS, but using TUS in that scenario is entirely optional, since S3 supports multipart file uploads.
